src/ethereum/api/Bridge.ts

```diff
@@ -41,13 +41,19 @@ export class Bridge {
   }
 
   static toFullItem(dbItem: ItemAttributes): FullItem {
-    return {
+    const fullItem = {
       ...dbItem,
+      urn: null,
       in_catalyst: false,
       is_approved: false,
       is_published: false,
       total_supply: 0,
     }
+
+    delete (fullItem as FullItem & { urn_suffix: string | undefined })
```

Comment:
Maybe we can use utils.omit or destructuring here and [here](https://github.com/decentraland/builder-server/pull/286/files#diff-770f618c954ffa5f7e17dad792dd6e16f45eb3c890e1f0616f18f7bd117af885R166) too?